### ISSUE TITLE
Set option for using different language

### ISSUE THREAD

**Issue by stefan-mueller** [kbenoit/spacyr#20]
A couple of months ago, German has been [added](https://spacy.io/docs/usage/) as a language to **spaCy** and more languages are planned (see also https://explosion.ai/blog/german-model). In Python, once I installed (`python -m spacy.en.download`), the German language model can be loaded by using `nlp = spacy.load('de')`. However, as far as I see **spacyr** seems to use English as the default and I did not find a command to switch between languages. 

If this is correct (maybe I just did not find this possibility), could we add an option to `spacy_initialize()` to switch between languages?

**Comment by kbenoit** [comment]
@amatsuo this is a good idea, to specify the language model when running `spacy_initialize()`.

**Comment by amatsuo** [comment]
@stefan-mueller I implemented the option. I believe that it is working properly, but since I don't understand the language, I cannot be sure. 

If you got a time, could you check it? You can first look at the test output here:
https://github.com/kbenoit/spacyr/tree/dev_lang_select#use-german-language-model

Also, you can install this branch by
```
devtools::install_github("kbenoit/spacyr", ref = 'dev_lang_select')
```
and check with other german texts you have.

**Comment by stefan-mueller** [comment]
@amatsuo , I checked it and it works perfectly. It even detects verb tenses. Thank you so much! I checked it with a variety of German party manifestos, and it works perfectly.

Just one minor aspect: The results of the German tagger are also summarised under the variable name `penn`. As far as I see, however, the parser used by **SpaCy** is the [German text archive](http://www.sfs.uni-tuebingen.de/resources/stts-1999.pdf) (see for an overview http://www.deutschestextarchiv.de/doku/pos). So maybe we should change `penn` to `stts`, the official name of this tagset? 

Thanks again for your help.

**Comment by kbenoit** [comment]
Interesting - is there a way to get the name of the tagset from the language model? 

The English trained set includes two tagsets, Penn and Google. I had not realised that each trained model could include others.

**Comment by stefan-mueller** [comment]
Good question, @kbenoit. Here is the best description of the German language model in **SpaCy** I could find: https://explosion.ai/blog/german-model.

I am pretty sure that the `penn`column currently shows the "German specific pos tags" (STTS). If you tell me where and how I can detect the exact name of the  German tagset in Python, let me know. I'm happy to help.

**Comment by amatsuo** [comment]
@stefan-mueller,
The exact place is L77 of `inst/python/initialize_rPython.py` where the code is:
```
        attr_name = "tag_" if tag_type == "penn" else "pos_"
```
So if an option `penn` is given, then `tag_` attribute is returned. Otherwise, `pos_` attribute is returned. In English model, `tag_` is penn tagset and `pos_` is google. Could you tell us what these attributes contain in German model?

**Comment by stefan-mueller** [comment]
Hi @amatsuo, I tried to find the attributes, but was not successful (so far I only used these packages and did not try to modify them…). I opened the `initialize_rPython.py`file which I found in my my R packages folder, but L77 was exactly as you described it above.  I copied the script, and run it in Python after changing `spacy.load(lang)`to `spacy.load('de')`. I'm sure I'm doing something completely wrong, but I don't know what I need to change to get the attributes of the German model. Sorry for this naive question, but I am just not able to figure this out.
